backend/api/simulation.py

```
@@ -3301,77 +3274,40 @@ async def get_simulation_grading(
-Output ONLY valid JSON, no extra text.
-"""
-        print(f"[PROMPT] LLM overall grading prompt:\n{prompt}")
+    if scene_feedback and learning_outcomes:
         try:
-            print(f"[DEBUG] LLM overall grading prompt: {prompt}")
-            response = client.chat.completions.create(
-                model="gpt-4o",
-                messages=[{"role": "user", "content": prompt}],
-                max_tokens=400,
-                temperature=0.2
+            print(f"[DEBUG] Using RAG-enabled grading agent for overall assessment")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)